x-pack/plugins/monitoring/dev_docs/reference/data_collection_modes.md

```diff
@@ -0,0 +1,51 @@
+Stack monitoring has a number of metrics collection modes:
+
+- Originally: [Internal collection](#internal-collection)
+- Currently recommended: [Standalone Metricbeat collection](#metricbeat-collection)
+- Planned future standard: [Package-driven collection](#package-collection)
+
+And just one log collection mode:
+
+- [Filebeat log collection](#filebeat-log-collection)
+
+## Metrics collection
+
+### Internal collection
+
+When using internal collection, each component in your Elastic stack will collect its own data via its own mechanism. In general this is an in-memory structure, but the implementation details vary per component.
+
+On a cycle (typically 10 seconds), each stack component then typically publishes the internally collected metrics to the [production deployment](../reference/terminology.md#production-deployment).
+
+The production cluster receives these metrics on the `/_monitoring/bulk` endpoint in elasticsearch, then forwards them to the [monitoring deployment](../reference/terminology.md#monitoring-deployment) along with its own metrics.
+
+Alternatively some stack components (logstash & beats) can be configured to send data directly to the [monitoring deployment](../reference/terminology.md#monitoring-deployment) either using a user-provided cluster UUID, or sending the data with no UUID to be associated with the [standalone cluster](../reference/terminology.md#standalone-cluster).
+
+### Metricbeat collection
+
+When using metricbeat collection, each component in your Elastic stack exposes an endpoint for metricbeat to collect metric data.
```

Comment:
When using standalone metricbeat collection

Reason: Agent uses metricbeat under the hood so there are times when we need to specify this difference so it's easier to just start with it, I think.